data/projects/implement_metrics_for_coala.md

```diff
@@ -7,28 +7,41 @@ them can be measured objectively by so-called
 offers a good base for the implementation or wrapping of algorithms that
 calculate these metrics from a piece of code.
 
-The goal of this project proposal should be to implement handling and
-display of software metrics in coala. To prove the concept, this project
-has to include multiple bears that calculate metrics for different
-programming languages.
+The goal of this project is to implement a module in coala that can produce
+software metrics because they are crucial when it comes to quality assurance.
+Source code metrics for quality assurance aims at writing maintainable code
+with less bugs. Picking right metrics is paramount and this project aims at
+implementing an API along with a set of Bears for handling and display of
+software metrics in coala. To prove the concept, a set of Bears implemented
+as a part of this project will calculate metrics for different programming
+languages.
```

Comment:
Instead of proving the concept I'd say the bears exist as a showcase, orientation and maybe tutorial

Also please address my "combinet he first three paragraphs" comment from last iteration.
Make it so that there is an intro paragraph with a Motivation (why do we want/need this project) and a goal and a second paragraph with technical information.